Rubberduck.VBEEditor/VBEHost/HostApplicationBase.cs

```diff
@@ -48,5 +48,13 @@ public TimeSpan TimedMethodCall(QualifiedMemberName qualifiedMemberName)
             stopwatch.Stop();
             return stopwatch.Elapsed;
         }
+
+        public void Dispose()
+        {
+            if (Application != null)
+            {
+                Marshal.ReleaseComObject(Application);
```

Comment:
Since it's a base class, it needs to implement the full Disposable pattern. https://msdn.microsoft.com/en-us/library/b1yfkh5e(v=vs.110).aspx